code/style.css

```diff
@@ -0,0 +1,153 @@
+body {
+  background: #e0e1dd;
+  color: #393d3f;
+  font-family: 'Athiti', Arial, Helvetica, sans-serif;
+}
+
+h1 {
+  margin: 20px;
+  font-size: 25px;
+  text-align: center;
+}
+
+button {
+  width: 150px;
+  height: 40px;
+  border-radius: 15px;
+  margin-bottom: 20px;
+  font-weight: bold;
+  background: #62929e;
+}
+
+.bot {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.bubble-container {
+  display: none;
+  align-items: flex-start;
+  justify-content: center;
+  width: 90%;
+  padding-right: 50px;
+  margin-bottom: 20px;
+}
+
+.bubble-start {
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  width: 90%;
+  padding-right: 50px;
+  margin-bottom: 20px;
+}
+
+.bot-img {
+  width: 30px;
+  margin-right: 10px;
+}
+
+.bubble {
+  flex-direction: column;
+  align-items: center;
+  width: 150px;
+  border-radius: 0 10px 10px 10px;
+  background: #c6c5b9;
+  padding: 20px;
+}
+
+.bubble-answer-container {
+  display: none;
+  align-items: flex-start;
+  justify-content: center;
+  width: 90%;
+  padding-left: 50px;
+  margin-bottom: 20px;
+}
+
+.user-img {
+  width: 25px;
+  margin-left: 10px;
+}
+
+.bubble-answer {
+  flex-direction: column;
+  align-items: center;
+  width: 150px;
+  border-radius: 10px 0 10px 10px;
+  background: #62929e;
+  padding: 20px;
+}
+
+input {
+  width: 132px;
+  margin-bottom: 10px;
+}
+
+select {
+  width: 140px;
+  padding: 1px 2px;
+  border-width: 2px;
+  border-style: inset;
+  margin-bottom: 10px;
+}
+
+.submit {
+  width: 70px;
+  margin-bottom: 0;
+}
+
+#nameForm {
```

Comment:
We shouldn't style by id's, it's always better to add class to that element as well and reference to it in CSS